src/components/Card/TransactionCard.tsx

```diff
@@ -5,6 +5,8 @@ import { Transaction, InputOutput } from '../../http/response/Transaction'
 import GreenArrowDown from '../../assets/green_arrow_down.png'
 import { startEndEllipsis } from '../../utils/string'
 import { shannonToCkb } from '../../utils/util'
+import PaginationList from '../Transaction/PaginationList'
+import { CELL_PAGE_SIZE } from '../Transaction'
```

Comment:
I suggest that we can put `CELL_PAGE_SIZE` to `TransactionCard`, because mobile page may be different from PC page and we should reduce coupling.

Reply:
Ok, I've fixed it. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/51/commits/8b226faf9c91c83a242f44f99ea688a6dd04e68f